src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/LocationGroupService.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services
+
+import uk.gov.justice.digital.hmpps.whereabouts.model.Location
+import uk.gov.justice.digital.hmpps.whereabouts.model.LocationGroup
+import java.util.function.Predicate
+
+interface LocationGroupService {
```

Comment:
Can we get away without an interface?

Reply:
It's actually appropriate in this case I think - there are 3 classes using the interface.

Reply:
Are there multiple implementations of it? If not would it be worth making the a actual type the dependency instead of the interface?

Reply:
Apologies if I've missed it.

Reply:
Spotted it.

Reply:
If you look at LocationGroupServiceSelector it implements the interface and orchestrates 2 other services that implement the interface.